media-libs/sdl2_mixer/sdl2_mixer-2.0.1.recipe

```diff
@@ -23,9 +23,10 @@ PROVIDES="
 	"
 REQUIRES="
 	haiku$secondaryArchSuffix
-	lib:libSDL2_2.0$secondaryArchSuffix
+	lib:libgl$secondaryArchSuffix
+	lib:libsdl2_2.0$secondaryArchSuffix
 	lib:libogg$secondaryArchSuffix
-	lib:libFLAC$secondaryArchSuffix
+	lib:libflac$secondaryArchSuffix
```

Comment:
Good catch! I had not noticed there was an unaesthetic-only case inconsistency in sdl2_mixer-2.0.1.recipe, **`lib:libFLAC`** in REQUIRES and **`devel:libflac`** in BUILD_REQUIRES.
I noticed, however, that there are aroud 7 recipes with **`lib:libflac`** and around 12 with **`lib:libFLAC`**. Since the name of the lib is libFLAC, wouldn't it be _sligthly_ better to change **`devel:libflac`** into **`devel:libFLAC`** (instead of changing **`lib:libFLAC`** to **`lib:libflac`**) ?
I do understand that all this does not make much difference, but I was wondering if there was a recommended or prefered spelling for **`lib:`** and **`devel:`** items for libs that have uppercase letters in their names.